test/pom.xml

```diff
@@ -97,6 +97,10 @@ THE SOFTWARE.
           <groupId>${project.groupId}</groupId>
           <artifactId>jenkins-war</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
better to fix upstream? easy to quickly integrate fix

Reply:
was about to say the same but jth should still be java 8 compatible for now right?

Reply:
currently till LTS I believe, if there's no fix we can do to it test harness I think we should file an issue against it and a TODO comment here?

Reply:
This is at least my qucik fix whilst I focus on getting a build of Jenkins so I can actually debug https://github.com/jenkinsci/jenkins/pull/6715#pullrequestreview-1023847744

jaxb was bundled with java8 so we may well be able to exclude it across the board.  (no idea how / if this impacts java17 as of yet though).  Will leave this here until I finish with the other issue.